manifests/metallb-frr.yaml

```diff
@@ -404,10 +404,13 @@ data:
   vtysh.conf: |+
     service integrated-vtysh-config
   frr.conf: |+
+    ! This file gets overriden the first time the speaker renders a config.
+    ! So anything configured here is only temporary.
     frr version 7.5.1
     frr defaults traditional
     hostname Router
     line vty
+    log file /etc/frr/frr.log informational
```

Comment:
This is gonna be overridden as soon as the speaker takes over right?

Reply:
Yes, as soon as the speaker makes the first reload due to a config change.